addons/web/static/src/views/form/form_label.xml

```diff
@@ -3,7 +3,7 @@
 
     <t t-name="web.FormLabel" owl="1">
         <label class="o_form_label" t-att-for="props.id" t-att-class="className" >
-            <t t-esc="props.string"/><sup class="btn-link p-1" t-if="hasTooltip" t-att="{'data-tooltip-template': 'web.FieldTooltip', 'data-tooltip-info': tooltipInfo, 'data-tooltip-touch-tap-to-show': 'true'}">?</sup>
+            <t t-esc="props.string"/><sup class="o_sup_tooltip btn btn-sm btn-link d-inline p-1" t-if="hasTooltip" t-att="{'data-tooltip-template': 'web.FieldTooltip', 'data-tooltip-info': tooltipInfo, 'data-tooltip-touch-tap-to-show': 'true'}">?</sup>
```

Comment:
Sorry for the headhake @mano-odoo .
This isn't a "button" actually, since it's not supposed to be clicked.

I think that the right style classes are `text-info p-1`

Reply:
🍰 _The `button` is a lie_

Indeed it's easier with the `text-info` class.